src/dbnode/storage/cleanup.go

```diff
@@ -402,11 +402,11 @@ func (m *cleanupManager) cleanupSnapshotsAndCommitlogs() (finalErr error) {
 	// Delete corrupt snapshot metadata files.
 	for _, errorWithPath := range snapshotMetadataErrorsWithPaths {
 		m.metrics.corruptSnapshotMetadataFile.Inc(1)
-		m.opts.InstrumentOptions().Logger().WithFields(
-			xlog.NewField("err", errorWithPath.Error),
-			xlog.NewField("metadataFilePath", errorWithPath.MetadataFilePath),
-			xlog.NewField("checkpointFilePath", errorWithPath.CheckpointFilePath),
-		).Errorf(
+		m.opts.InstrumentOptions().Logger().With(
+			zap.Error(errorWithPath.Error),
```

Comment:
Same here with `zap.Error()` at the end.

Reply:
done